topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -410,7 +420,7 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    1. `admin_users` to the email address you will use with this Galaxy.
 >    2. `brand` to something fun!
 >    3. `database_connection` to point to the database you setup earlier (`postgresql:///galaxy?host=/var/run/postgresql`).
->    4. `file_path` to a place to store data, `/data` for this lesson.
+>    4. `file_path` to a place to store data, `/data` is fine for this lesson which sets up a single-node Galaxy. If you have separate compute machines, this will need to be storage shared between the galaxy node and compute nodes.
```

Comment:
```suggestion
>    4. `file_path` to a place to store data, `/data` is fine for this lesson which sets up a single-node Galaxy. If you have separate compute machines, this will normally need to be storage shared between the Galaxy node and compute nodes.
```

to consider the Pulsar setup too.

Reply:
Thanks, def a better phrasing!